src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/BotSegments/Models/CVObjectDetectionMouseActionData.cs

```diff
@@ -0,0 +1,365 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+using RegressionGames.StateRecorder.BotSegments.JsonConverters;
+using RegressionGames.StateRecorder.BotSegments.Models.CVService;
+using RegressionGames.StateRecorder.JsonConverters;
+using RegressionGames.StateRecorder.Models;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace RegressionGames.StateRecorder.BotSegments.Models
+{
+    /**
+     * <summary>Data for clicking on or moving the mouse to a CV Object Detection location in the scene</summary>
+     */
+    [Serializable]
+    [JsonConverter(typeof(CVObjectDetectionMouseActionDataJsonConverter))]
+    public class CVObjectDetectionMouseActionData : IBotActionData
```

Comment:
I feel like these should be CVObjectDetectionText to better align with the BotActionType

Reply:
This name doesn't have 'Text' specifier because the same script is used for Object Detection with Image query as well.